WordPress/src/jetpack/java/org/wordpress/android/ui/accounts/login/LoginPrologueRevampedViewModel.kt

```diff
@@ -73,7 +73,9 @@ class LoginPrologueRevampedViewModel @Inject constructor(
 
         override fun onSensorChanged(event: SensorEvent?) {
             event?.values?.let { (_, yAxisAcceleration, _) ->
-                acceleration = yAxisAcceleration * ACCELERATION_FACTOR + DRIFT
+                acceleration = yAxisAcceleration * ACCELERATION_FACTOR +
+                        (if (yAxisAcceleration >= -0.2f) DRIFT else -DRIFT) // drift shouldn't change scroll direction.
+
```

Comment:
Can you elaborate on the goal of this change?

The original purpose I had in mind for the `DRIFT` constant was to provide a slight "force" (i.e. biasing the acceleration) so that when the y component of the acceleration vector was zero (i.e. device flat on a table), the text would still scroll.

Is the intention of this change to remove the stability region when the acceleration exactly "cancels" the drift (e.g. when the device is in a "declined" position, tilted slightly backward)? Afaict, this change introduces a discontinuity in the output for a continuous input.

From earlier discussions, I'm thinking that this wasn't the most appropriate place to put this `DRIFT`, and that rather than baking it into the `onSensorChanged` function, it would be better to be moved to the `updateForFrame` function, which is more concerned with updating the physics model. Wdyt?

Reply:
> Afaict, this change introduces a discontinuity in the output for a continuous input.

Good point! this makes it far from ideal but with the latest adjustments to `ACCELERATION_FACTOR` & `VELOCITY_DECAY` it seemed like a needed thing.

The picture below illustrates the issue it's fixing:

<img width="813" alt="pose_motion_dir" src="https://user-images.githubusercontent.com/4588074/193809700-721871b4-811d-41a5-8f9d-26388bdc6d7e.png">

At the device pose from the image we'd expect the scroll direction to be reversed, this was not the case since the `DRIFT` is a constant value added on top of the sensor data.

To explain a bit more why I adjusted the acceleration factor and velocity decay we have to take into consideration the `MAXIMUM_VELOCITY`.
This value is constraining the "responsiveness" of the physics, since any device rotation that outputs a value outside of this **max limit**, either positive or negative, will not be reflected on the scrolling speed.

before starting to make adjustments, I've added logging for the `velocity` and played with the `Virtual sensors` → `Device Pose` on the emulator.
This way I noticed with numbers what felt to be the visible with the eye: the device rotation is only affecting the speed of the animation when we're almost near the "flat on the table" device pose. Anything above this has no effect on the speed.

<img width="813" alt="Screenshot 2022-10-04 at 13 49 21" src="https://user-images.githubusercontent.com/4588074/193812156-551d47a5-40c1-4531-b577-844da3429514.png">

As seen above, any vertical rotation above that pose has no effect on the animation speed.

While playing with the values I figured out lowering the `ACCELERATION_FACTOR` results in more responsiveness, I.e. The rotation threshold needed to hit the `MAXIMUM_VELOCITY` can be adjusted so that we have different `velocity` over the entire rotation range from full vertical to full horizontal:

<img width="1155" alt="Screenshot 2022-10-04 at 13 55 22" src="https://user-images.githubusercontent.com/4588074/193813316-f63976d3-a30b-46d4-9516-a905e4f187c2.png">

In the image above we have the current values `ACCELERATION_FACTOR = -0.015f` instead of `-0.1f`.
Because of this, the velocity starts to change from a rotation of `-18` compared to `-83`.

Reply:
Thanks a lot for exploring the different approach for sensors motion @mkevins 🙇 

I opened a PR targeting this branch from your work 🎉  : https://github.com/wordpress-mobile/WordPress-Android/pull/17265